python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -219,6 +220,7 @@ def quantize(self, model,  # remove the type requirement for type checking
                  precision='int8',
                  accelerator=None,
                  calib_dataloader: DataLoader = None,
+                 inference_method_name = "forward",
```

Comment:
move this argument down in the list, say after `val_dataloader` or after `accuracy_criterion`

Reply:
Done